### PR TITLE
fix: fix fork application spec

### PR DESCRIPTION
**Pull request by dipyamanbiswas07**
## Description

Due to the removal of the Templates tab, the fork_templates spec file needs to be modified in according with the onboarding flow. Reconnect Datasource test is moved to a more appropriate spec file.

#### PR fixes following issue(s)
Fixes # (issue number)
> if no issue exists, please create an issue and ask the maintainers about this first
>
>
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Chore (housekeeping or task changes that don't impact user perception)
- This change requires a documentation update
>
>
>
- [ ] Manual
- [ ] JUnit
- [ ] Jest
- [ ] Cypress
>
>
#### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

File: app/client/cypress/e2e/Regression/ClientSide/Templates/Fork_Template_spec.js [hunk 1]
```diff
@@ -16,78 +16,11 @@ describe(
...
-        "Internal server error while processing request",
-      );
-      _.agHelper.AssertElementExist(templateLocators.templateCard);
-      _.agHelper.AssertElementAbsence(templateLocators.templateForkButton);
-      _.agHelper.GetNClick(templateLocators.templateCard);
-      _.agHelper.AssertElementExist(templateLocators.templateCard);
-      _.agHelper.AssertElementAbsence(templateLocators.templateViewForkButton);
-    });
-
-    it("4. Check if tooltip is working in 'Reconnect Datasources'", () => {
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Templates/Fork_Template_spec.js:63]: Please alter spec accordingly. Cant be removed,

Comment (dipyamanbiswas07) [reply app/client/cypress/e2e/Regression/ClientSide/Templates/Fork_Template_spec.js:63]: Moved it to a different spec

File: app/client/cypress/e2e/Regression/ClientSide/Templates/Filtering/TemplatesModal_filtering_spec.ts [hunk 2]
```diff
@@ -39,5 +41,27 @@ describe(
...
         .should((cards) => expect(cards.length).to.be.equal(TEMPLATES_COUNT));
     });
+    it("2. Check if tooltip is working in 'Reconnect Datasources", () => {
+      PageList.AddNewPage("Add page from template");
+      agHelper.AssertElementVisibility(templates.locators._templateDialogBox);
+
+      cy.xpath("//h1[text()='Customer Messaging Tool']")
+        .scrollIntoView()
+        .wait(500)
+        .click();
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Templates/Filtering/TemplatesModal_filtering_spec.ts:51]: USe agHelper.GetNClick.

Comment (dipyamanbiswas07) [reply app/client/cypress/e2e/Regression/ClientSide/Templates/Filtering/TemplatesModal_filtering_spec.ts]: Done

File: app/client/cypress/e2e/Regression/ClientSide/Templates/Fork_Template_spec.js [hunk 3]
```diff
@@ -16,78 +16,11 @@ describe(
       _.agHelper.GetNClick(templateLocators.startFromTemplateOnboardingCard);
     });
 
-    it("1. Fork a template to an workspace", () => {
+    it("1. Fork a template to an workspace during onboarding should open the template in an application", () => {
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Templates/Fork_Template_spec.js:19]: FYI - I have made changes to title in EE. Need to resolve it carefully.

**Review by Aishwarya-U-R — APPROVED** [review]

**Comment by Aishwarya-U-R** [comment]
@dipyamanbiswas07 - please fix the description of the PR. 1-2 lines about why tests removed & about test move.

**Pull request merged**